guava/src/com/google/common/util/concurrent/Uninterruptibles.java

```diff
@@ -93,6 +95,62 @@ public static boolean awaitUninterruptibly(CountDownLatch latch, long timeout, T
     }
   }
 
+  /** Invokes {@code condition.}{@link Condition#await() await()} uninterruptibly. */
+  @GwtIncompatible // concurrency
+  public static void awaitUninterruptibly(Condition condition) {
```

Comment:
@cpiotr I think this method can be removed, cause it adds no value.

Reply:
@ChristianSchwarz Good point, removed.